packages/common-all/src/dnode.ts

```diff
@@ -334,6 +334,14 @@ export class DNodeUtils {
   static getLeafName(note: DNodeProps) {
     return _.split(note.fname, ".").pop();
   }
+
+  static cleanFname(fname: string) {
+    if (_.isUndefined(fname) || fname === "null") return;
```

Comment:
is there every a time when `fname` will equal the "null" string? if not, this should use the `_.isNull` check instead. currently, this results in an exception when I run the importPod because of the following entry:

{"ctx":"membersToNotes","name":"","github":null,"discord":null,"member":{"custom":{"orbitId":"GZSYO0vz","social":{"linkedin":null,"github":null,"twitter":"le__poulpe","discord":null}}},"noteName":null,"msg":"found null"}

Reply:
made changes for this in latest commit. added twitter and email in fname fallback.